source/extensions/priv/server/precomp.h

```diff
@@ -2,16 +2,23 @@
 #define METERPRETER_SOURCE_EXTENSION_PRIV_SERVER_PRECOMP_H
 
 #define  _WIN32_WINNT 0x0400
+#define JET_VERSION 0x0501 

```

Comment:
I don't think there is much reason to put defines and includes specific to one C file into precomp.h. Shouldn't this really consist only of common headers between all of the C files in the extension? In the past, I've only really even used precompiled headers to speed up heavy template instantiations with C++, something that's not even happening here.

This also changes a few source file privileges from 644 to 755. It would be nice to leave them non-executable.